services/cartService.js

```diff
@@ -0,0 +1,13 @@
+const cartDao = require("../models/cartDao");
+
+const cartList = async(userId) => {
+    const list = await cartDao.productList(
+        userId
+    );
+
+    return list;
+};
```

Comment:
한 줄로 줄일 수 있을 것 같습니다.
일반적으로 한 줄에 문자가 80 이하인 경우에는 한 줄로 표현하는 것이 가독성이 좋습니다.
```suggestion
const getCarts = async (userId) => await cartDao.productList(userId)
```